weave

```diff
@@ -2146,6 +2153,7 @@ EOF
         ;;
     reset)
         [ $# -eq 0 ] || usage
+        remove_plugin_network || echo "WARNING: docker may hang in this state; re-launch weave and remove active endpoints before shutting down" >&2
```

Comment:
What does 'remove active endpoints' look like in terms of docker commands? I suspect this phrase will be meaningless to most users